source/image/ImageBuffer.h

```diff
@@ -15,8 +15,10 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #pragma once
 
+#include "ImageFileData.h"
```

Comment:
You can forward-declare this

Reply:
I'd have to import it in a lot of other files, though.